src/App.test.js

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```

Comment:
Deleting this can effect the tests run by TravisCI. It's probably better to keep this and edit the travis.yml file so it runs the test on the client directory for now.

Reply:
the script section of travis.yml should start with
`- cd client`